docker/Dockerfile.ubuntu.xenial

```diff
@@ -0,0 +1,35 @@
+# Dockerfile for building an image with ubuntu xenial
+FROM sociomantictsunami/base
+MAINTAINER andreas-bok-sociomantic <andreas.bok@sociomantic.com>
+
+WORKDIR /root
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# COPY ./container_entrypoint.sh ./install_dependencies.sh /usr/local/bin/
+
+# RUN chmod +x /usr/local/bin/container_entrypoint.sh
+# RUN chmod +x /usr/local/bin/install_dependencies.sh
+
+# delete all the apt list files since they're big and get stale quickly
+RUN rm -rf /var/lib/apt/lists/*
+# this forces "apt-get update" in dependent images, which is also good
+
+# enable the universe
+RUN sed -i 's/^#\s*\(deb.*universe\)$/\1/g' /etc/apt/sources.list
+
+# Add repo for llvm clang
+RUN echo "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial main" >> /etc/apt/sources.list
+RUN echo "deb-src http://apt.llvm.org/xenial/ llvm-toolchain-xenial main" >> /etc/apt/sources.list
+
+# install apt-fast to download required packages in parallel
+RUN echo "deb http://ppa.launchpad.net/saiarcot895/myppa/ubuntu xenial main" >> /etc/apt/sources.list
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys DC058F40
+RUN apt-get update
+RUN apt-get install --no-install-recommends -y apt-fast
+
+# install required packages
+RUN apt-fast -y install --no-install-recommends \
+    libssl-dev autoconf libtool make automake zlib1g-dev
```

Comment:
Every `RUN` command in Docker creates a new layer, it is usually recommended to create some script and just use one `RUN` (or if it's just a few commands use `RUN cmd && cmd && cmd`). Not a big deal, just FYI.

Reply:
Oki, you mean a new temp image will be created for each RUN cmd? 

I will wrap the cmds  in a script

Reply:
Yup